src/docgen/schema.ts

```diff
@@ -493,7 +497,16 @@ export function extractDocs(docs: jsii.Docs): DocsSchema {
   });
 }
 
-function filterUndefined<T>(obj: T): T {
+/**
+ * Generates the name of the submodule.
```

Comment:
Is this really extracting the name? The function seems to suggest it extracts path, and its usage suggest it extract an FQN.

Can you align all these? I think `submodulePath` actually correctly represents what this function is doing, but in that case I would change the schema to declare `submodulePath` as well, not `submoduleFqn`.